Library/Formula/voikko-fi.rb

```diff
@@ -0,0 +1,26 @@
+class VoikkoFi < Formula
```

Comment:
So... you'd change build dependency to foma to runtime dependency so that you could install voikko-fi as a resource instead of separate package? What about head build, is that possible with resource?